test/unit/testIhtValue.js

```diff
@@ -14,27 +14,41 @@ describe('IhtValue', () => {
         });
     });
 
-    describe('handlePost()', () => {
-        let ctx;
-        let errors;
-        let formdata;
-        let session;
-        let hostname;
-        let featureToggles;
-
-        it('should return the ctx with the deceased married status and the screening_question feature toggle', (done) => {
-            ctx = {
+    describe('getContextData()', () => {
+        it('should return the ctx with the deceased address and the screening_question feature toggle on', (done) => {
+            const req = {
+                sessionID: 'dummy_sessionId',
+                session: {form: {}, featureToggles: {screening_questions: true}},
+                body: {
+                    grossValueOnline: '500000',
+                    netValueOnline: '400000'
+                }
+            };
+            const ctx = IhtValue.getContextData(req);
+            expect(ctx).to.deep.equal({
                 grossValueOnline: '500000',
-                netValueOnline: '400000'
+                netValueOnline: '400000',
+                isToggleEnabled: true,
+                sessionID: 'dummy_sessionId'
+            });
+            done();
+        });
+
+        it('should return the ctx with the deceased address and the screening_question feature toggle off', (done) => {
```

Comment:
Please update the description for this test as it is for iht value but it is referring to deceased address.